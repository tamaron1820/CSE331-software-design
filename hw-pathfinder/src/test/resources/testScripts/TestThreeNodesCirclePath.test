# Testing Dijkstra's algorithm when there are three nodes which are connected as circle

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 5.0
AddEdge graph1 n2 n3 5.0
AddEdge graph1 n3 n1 5.0

FindPath graph1 n1 n2

FindPath graph1 n2 n3

FindPath graph1 n3 n1

FindPath graph1 n1 n3