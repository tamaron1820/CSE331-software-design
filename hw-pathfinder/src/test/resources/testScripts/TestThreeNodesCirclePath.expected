# Testing Dijkstra's algorithm when there are three nodes which are connected as circle

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 5.000 from n1 to n2 in graph1
added edge 5.000 from n2 to n3 in graph1
added edge 5.000 from n3 to n1 in graph1

path from n1 to n2:
n1 to n2 with weight 5.000
total cost: 5.000

path from n2 to n3:
n2 to n3 with weight 5.000
total cost: 5.000

path from n3 to n1:
n3 to n1 with weight 5.000
total cost: 5.000

path from n1 to n3:
n1 to n2 with weight 5.000
n2 to n3 with weight 5.000
total cost: 10.000